COMPILING.md

```diff
@@ -77,7 +79,13 @@ Install git, make, cmake and clang (or gcc), using your platform's package manag
 ```
 sudo apt-get install git make cmake clang
 ```
-Ensure that you have modern C++ compiler and linker (Clang 3.5+, GCC 4.8+, or VS 2013+).
+Ensure that you have modern C++ compiler and linker (Clang 7.0+, GCC 7.4+, or VS 2015+).
```

Comment:
omg, it's too late

Reply:
It should be 2017

Reply:
If 2017 isn't in CI then we don't support it.

Reply:
i'm developing on 2017, I'd be the first to complain if it didn't work

Reply:
Then why use 2019 in CI?

Reply:
2019 has a couple of codegen improvements (e.g. reduced binary sizes for code with exceptions) and since the Windows builds are being served by the AppVeyor artefacts, we certainly wish to give our users the bestest code ;)

Reply:
That's a good point, but it's a bit unfortunate. Could we use 2017 for the debug build?

Reply:
I don't know how to do it on AppVeyor. In any case we're targeting a language version, not a compiler. I tell cmake that 17 is required, if it passes I'm happy. If anything we need to build on the latest release, especially given that newer ones have more warnings that will fail the build.